runner/context.go

```diff
@@ -79,22 +92,49 @@ func (c *ContainerContext) parseVolumes(cargs []string) {
 			vols = append(vols, expandVolumeString(strings.TrimSpace(strings.TrimPrefix(v, "--volume"))))
 			continue
 		}
+
+		newCargs = append(newCargs, v)
 	}
 	c.WithVolumes(vols...)
+
+	return newCargs
 }
 
-func (c *ContainerContext) parseUserArgs(cargs []string) {
+func (c *ContainerContext) parseUserArgs(cargs []string) ([]string, error) {
+	newCargs := []string{}
+	seenUserArg := false
+
 	for _, v := range cargs {
 		v = os.ExpandEnv(strings.TrimSpace(v))
 		if strings.HasPrefix(v, "-u") {
+			if seenUserArg {
+				return nil, fmt.Errorf("error in container_args, user flag (-u/--user) already seen. Found: %s", v)
+			}
 			c.user = strings.TrimSpace(strings.TrimPrefix(v, "-u"))
-			break
+			seenUserArg = true
+			continue
```

Comment:
is the idea that we notify the user of incorrectly added duplicate keys as opposed silently taking only the first user arg

Reply:
Yeah